src/Events/Events.js

```diff
@@ -132,42 +132,46 @@ class Events {
     /**
      * @param {string} eventKey
      * @param {?string} objectId
+     * @param {?object} params
      * @returns {AsyncIterator}
      */
-    statusChanges(eventKey, objectId = null) {
+    statusChanges(eventKey, objectId = null, params = {}) {
```

Comment:
I went with `StatusChangesParams` because that's how we did Charts and Subaccounts, so just for consistency. For the docs, yeah it seems like I forgot it for `statusChanges` method but the others should be good, I will fix that